store/src/sqlite/model.rs

```diff
@@ -1154,13 +1134,53 @@ mod test {
         // No keys in range
         let ret = recon::Store::first_and_last(
```

Comment:
The comment seems to indicate that there should be no keys in range, but it looks like the test now returns some?

Reply:
fixed